js/source/image_source.js

```diff
@@ -31,7 +31,7 @@ module.exports = ImageSource;
  * map.removeSource('some id');  // remove
  */
 function ImageSource(options) {
-    this.urls = options.urls;
+    this.url = options.url;
```

Comment:
I'm pretty sure this is just a bugfix - sure, we'd serialize urls instead of url, but you couldn't rely on that behavior at all because ajax.getImage would immediately fail because of that mistake.